CHANGELOG.md

```diff
@@ -78,4 +78,4 @@
 
 #### Mix
 
-  * [Mix.Utils] `underscore/1` and `camelize/1` are deprecated
```

Comment:
Is this change killing the extra newline at the end of this file? Never seen a diff like this but just checked CHANGELOG.md and indeed there's an extra newline at the end of the file :\